source/PreferencesPanel.cpp

```diff
@@ -512,7 +513,11 @@ void PreferencesPanel::DrawSettings()
 		"Show escort systems on map",
 		"Show stored outfits on map",
 		"System map sends move orders",
-		ALERT_INDICATOR
+		"Warning siren",
```

Comment:
```suggestion
```
why are you adding this?

Reply:
probably just a merge thing

Reply:
yeah, that popped out as part of the merge... I'll try to rebase the branch and fix